specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2264 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Spacecraft"
+          },
+          "description": "A list of spacecraft resources in a resource group."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for the ListSpacecrafts API service call."
+    },
+    "SpacecraftLink": {
+      "type": "object",
+      "required": [
+        "name",
+        "centerFrequencyMHz",
+        "bandwidthMHz",
+        "direction",
+        "polarization"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Link name.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "centerFrequencyMHz": {
+          "type": "number",
+          "description": "Center Frequency in MHz.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "bandwidthMHz": {
+          "type": "number",
+          "description": "Bandwidth in MHz.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "direction": {
+          "type": "string",
+          "description": "Direction (uplink or downlink).",
+          "enum": [
+            "uplink",
+            "downlink"
+          ],
+          "x-ms-enum": {
+            "name": "direction",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "polarization": {
+          "type": "string",
+          "description": "Polarization. e.g. (RHCP, LHCP).",
+          "enum": [
+            "RHCP",
+            "LHCP",
+            "linearVertical",
+            "linearHorizontal"
+          ],
+          "x-ms-enum": {
+            "name": "polarization",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "authorizations": {
+          "type": "array",
+          "description": "Authorized Ground Stations",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/AuthorizedGroundstation"
+          },
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "List of authorized spacecraft links per ground station and the expiration date of the authorization."
+    },
+    "Spacecraft": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SpacecraftsProperties",
+          "description": "Spacecraft Properties",
+          "allOf": [
+            {
+              "$ref": "#/definitions/SpacecraftsProperties"
+            }
+          ]
+        },
+        "etag": {
```

Comment:
We don't actually expose an etag. Etags look like they are optional: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#etags-for-resources

We might want to consider removing this from the swagger for now since we're not actually returning this.

Reply:
Yes, you shouldn't document things that you don't actually return. This will probably trigger breaking change validations. They should be okay to approve it once they get the explanation that its actual service behavior.